ci/travis/linux/install.sh

```diff
@@ -1,5 +1,12 @@
 mkdir build
 cd build
-cmake -DWITH_SERVER=ON -DWITH_STAGED_PLUGINS=OFF -DWITH_GRASS=OFF \
+
+CMAKE_OPTS = -DWITH_SERVER=ON -DWITH_STAGED_PLUGINS=OFF -DWITH_GRASS=OFF \
           -DSUPPRESS_QT_WARNINGS=ON -DENABLE_MODELTEST=ON -DENABLE_PGTEST=ON \
-          -DWITH_QWTPOLAR=OFF -DWITH_APIDOC=ON -DWITH_PYSPATIALITE=ON ..
+          -DWITH_QWTPOLAR=OFF -DWITH_APIDOC=ON -DWITH_PYSPATIALITE=ON
+
+if [${QT} == 5]; then
```

Comment:
oh yeah...
anyway I am afraid that it will be hard to really do.
Basically because of QWT and QScintilla which I think are not available as Qt5 binaries.
Either yet another switch WITH_QWT and WITH_QSCINTILLA or packaging them for an old ubuntu version...